src/portable_python/cpython.py

```diff
@@ -107,7 +107,7 @@ def url(self):
         return f"https://www.python.org/ftp/python/{self.version.main}/Python-{self.version}.tar.xz"
 
     def xenv_LDFLAGS_NODIST(self):
-        yield f"-L{self.deps_lib}"
+        yield from (f"-L{lib_dir}" for lib_dir in self.deps_lib_dirs)
```

Comment:
I see, so multiple times this appears :)

Yeah, so nvm then... no great way to factor this :)